controllers/territory_controller/territory_controller.py

```diff
@@ -119,7 +119,11 @@ def is_locked(self, station_code: StationCode) -> bool:
         return station_code in self._locked_territories
 
     def get_claim_count(self, station_code: StationCode) -> int:
-        return len([x for x, _, _, _ in self._log if x == station_code])
+        return len([
+            x
+            for x, y, _, _ in self._log
```

Comment:
Descriptive variable names here would be good!

Reply:
eae94f394851bd0c2ceb536b4fcfea4d4fd94b6a